src/cpacs_other/CTiglUIDManager.h

```diff
@@ -67,13 +67,13 @@ class CTiglUIDManager
     TIGL_EXPORT TypedPtr ResolveObject(const std::string& uid, const std::type_info& typeInfo) const;
 
     template<typename T>
-    T& ResolveObject(const std::string& uid) const
+    TIGL_EXPORT T& ResolveObject(const std::string& uid) const
     {
         return *static_cast<T* const>(ResolveObject(uid, typeid(T)).ptr);
     }
 
     template<typename T>
-    std::vector<T*> ResolveObjects() const
+    TIGL_EXPORT std::vector<T*> ResolveObjects() const
```

Comment:
Please remove this export. This is a inline method which must not be exported.

Reply:
done